docs/source/user/custom_css.rst

```diff
@@ -0,0 +1,105 @@
+.. Copyright (c) Jupyter Development Team.
+.. Distributed under the terms of the Modified BSD License.
+
+Applying Custom CSS
+-------------------
+
+To apply custom CSS, you can add a ``/custom/custom.css`` file in the
+jupyter ``config`` directory. You can find the path, ``~/.jupyter``, to
+this directory by running ``jupyter --paths``. There you can create a
+folder named ``custom`` and create a ``custom.css`` file within the
+folder.
+
+Custom CSS is loaded by default. In the jupyter configuration directory, the
+``~/.jupyter/custom/custom.css`` file will be loaded unless the the application is
+initialized with the ``custom_css`` flag with the argument set to ``False`` as in
+``--LabApp.custom_css=False``.
+
+Jupyter Styling
+---------------
+
+You can use a custom CSS file to modify default Jupyter styling.
+
+::
+
+   /* Modify Jupyter Styles */
+
+  .lm-BoxPanel-child, .jp-FileBrowser-Panel, .lm-StackedPanel-child, .jp-FileBrowser-toolbar {
+      background-color: #aecad4!important;
+  }
+
+  #top-panel-wrapper, #jp-top-bar {
+      background-color: #aecad4!important;
+  }
+
+  #menu-panel-wrapper, #jp-MainMenu, #menu-panel {
+      background-color: #aecad4!important;
+  }
+
+  .jp-NotebookPanel-toolbar {
+      background-color:  #aecad4!important;
+  }
+  .lm-MenuBar-content {
+      color: #02484d
+  }
+
+  .lm-TabBar-content, .jp-DirListing-content, .lm-MenuBar-content {
+      font-size: small;
+  }
+
+.. figure:: https://user-images.githubusercontent.com/12378147/248197127-7e825096-3394-438f-9136-be4cc5c5cf01.png
```

Comment:
It would be great if we could generate these figures using playwright (galata) rather than rely on githubusercontent.com domain. It would also ensure that no browser UI is captured and that we are actually testing the integration works